### PR TITLE
[DO NOT MERGE] Adds Heroku deployment script

### PR DESCRIPTION
**Pull request by sopankhosla**
Custom config variables from Heroku are yet to be added. So please do not merge.

Issue is here: https://github.com/loklak/loklak_webclient/issues/617

### CODE REVIEW

**Comment by mariobehling** [comment]
Any update?

File: install_heroku_script.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+cd iframely/
+npm install
+echo "---------------1-------------------"
+cd ../
+cd oauth-proxy/
+npm install
+echo "------------------------------2--------------------------"
+cd ../
+gulp dev
```
Comment (sudheesh001) [inline install_heroku_script.sh]: heroku deployments, `gulp prod` and then the generated build needs to be served or maybe `gulp live`

Comment (sopankhosla) [reply install_heroku_script.sh]: @sudheesh001 could you please elaborate a bit more. I am not sure if I follow you here!

File: install_heroku_script.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+cd iframely/
+npm install
+echo "---------------1-------------------"
+cd ../
+cd oauth-proxy/
+npm install
+echo "------------------------------2--------------------------"
+cd ../
+gulp prod
```
Comment (sudheesh001) [inline install_heroku_script.sh:9]: `gulp dev` contains unminified javascript files, good for development and debugging takes time to load
`gulp prod` creates the files for being hosted on a web server
`gulp live` runs the web server and the application in an environment between dev and prod, minifies files

**Comment by mariobehling** [comment]
I am merging this. Please help improve it step by step. Thank you.

**Pull request merged**